src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1516,14 +1529,16 @@ InferenceProfiler::DetermineStatsModelVersion(
       *status_model_version = std::stoll(model_identifier.second);
     }
   }
-
-  // In case of ensemble models, if top level response caching is enabled,
-  // the composing models versions are unavailable in case of a cache hit.
-  // This is due to the scheduler sends cache response and composing models do
-  // not get executed. It's a valid scenario and shouldn't throw error.
-  bool is_model_version_specified =
+  // FIXME - Investigate why composing model version is -1 in case of ensemble
```

Comment:
Usually good practice to include the ticket number if you have one for follow-up like
```
// FIXME [DLIS-XXXX]: ...
```